spec/unit/installer_spec.rb

```diff
@@ -184,12 +184,21 @@ def @installer.write_pod_project
           pod 'OrangeFramework', :path => (fixture_path + 'orange-framework').to_s
           pod 'monkey',          :path => (fixture_path + 'monkey').to_s
         end
-        lockfile = generate_lockfile
-        config.integrate_targets = false
+        @lockfile = generate_lockfile
+      end
 
-        @installer = Installer.new(config.sandbox, podfile, lockfile)
+      it 'detects transitive static dependencies which are linked directly to the user target' do
+        Sandbox::FileAccessor.any_instance.stubs(:vendored_libraries).returns([Pathname('libThing.a')])
+        @installer = Installer.new(config.sandbox, @podfile, @lockfile)
         should.raise(Informative) { @installer.install! }.message.should.match /transitive.*libThing/
       end
+
+      it 'allows transitive static dependencies which contain other source code' do
+        Sandbox::FileAccessor.any_instance.stubs(:source_files).returns([Pathname('/yolo.m')])
```

Comment:
Because the `install!` is actually running further in this spec, it needs to be absolute to not fail. Let's change the other spec to also be absolute?